packages/notifi-react-card/lib/components/subscription/NotifiBellIcon.tsx

```diff
@@ -0,0 +1,83 @@
+import clsx from 'clsx';
+import React, { useEffect, useRef, useState } from 'react';
+import { PropsWithChildren } from 'react';
+
+import { BellIcon } from '../../assets/BellIcon';
+
+type Props = { color: string };
+
+type NotifiBellIconProps = {
+  svgProps?: Props;
+  bellColor?: string;
+  classNames?: Readonly<{
+    NotifiBellIcon: string;
+  }>;
+  containerStyles?: React.CSSProperties;
```

Comment:
I don't think we expose styles directly elsewhere, do we? We should retain convention and expose separate classNames for the other stuff